samples/notebooks/prediction/SDK_Custom_Predict_SDK_Integration.ipynb

```diff
@@ -0,0 +1,1311 @@
+   "metadata": {
+    "id": "51e149fdec1b"
+   },
+   "source": [
+    "## Build and push container to Artifact Registry"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "3bdb9a7768a5"
+   },
+   "source": [
+    "### Build your container"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "8phJWBhe7u9u"
+   },
+   "source": [
+    "#### Set up credentials (Optional)\n",
+    "\n",
+    "Setting up credentials is only required to run the custom serving container locally.\n",
+    "\n",
+    "First enable the IAM API if it's not already enabled."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "mERTlLb6dluB"
+   },
+   "outputs": [],
+   "source": [
+    "!gcloud services enable iam.googleapis.com"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "sBCra4QMA2wR"
+   },
+   "source": [
+    "Follow these steps:\n",
+    "\n",
+    "1. In the Cloud Console, go to the [**Create service account key**\n",
+    "   page](https://console.cloud.google.com/apis/credentials/serviceaccountkey).\n",
+    "\n",
+    "2. Click **Create service account**.\n",
+    "\n",
+    "3. In the **Service account name** field, enter a name, and\n",
+    "   click **Create**.\n",
+    "\n",
+    "4. In the **Grant this service account access to project** section, click the **Role** drop-down list. Type \"Vertex AI\"\n",
+    "into the filter box, and select\n",
+    "   **Vertex AI Administrator**. Type \"Storage Object Admin\" into the filter box, and select **Storage Object Admin**.\n",
+    "\n",
+    "Next, generate the service account key, and save it to the `credentials.json` path."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "f4lduXfvdluB"
+   },
+   "outputs": [],
+   "source": [
+    "SERVICE_ACCOUNT = \"[SERVICE_ACCCOUNT]\"  # @param {type:\"string\"}\n",
+    "\n",
+    "!gcloud iam service-accounts keys create credentials.json --iam-account=$SERVICE_ACCOUNT"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "240578ec9efe"
+   },
+   "source": [
+    "#### Build your custom container\n",
+    "\n",
+    "To build a custom image, a Dockerfile is necessary where you need to implement what the image looks like. However, with the Custom Prediction Routine features, Vertex SDK also generates Dockerfile and build images for you.\n",
+    "\n",
+    "Using `python:3.7` as a base image by default."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "3d3a6b9ed22b"
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "from google.cloud.aiplatform.prediction import LocalModel\n",
+    "\n",
+    "from src_dir.predictor import CprPredictor  # Update this path as the variable $USER_SRC_DIR to import the custom predictor.\n",
+    "\n",
+    "local_model = LocalModel.create_cpr_model(\n",
```

Comment:
I really love how much simpler the interface is for users :)

Reply:
: )